libioc/Config/Jail/Defaults.py

```diff
@@ -94,7 +94,7 @@
     "mount_procfs": "0",
     "mount_devfs": "1",
     "mount_fdescfs": "0",
-    "securelevel": "2",
+    "securelevel": 2,
```

Comment:
The `mount_*` BaseConfig items relate to usr.sbin/jail arguments, not to jail params. Handling of them remains unchanged, so that a change is not required in terms of introducing py-freebsd_sysctl